promscale/send-data/zipkin.md

```diff
@@ -0,0 +1,46 @@
+# Send Zipkin traces to Promscale
+Promscale natively supports the OpenTelemetry Line Protocol (OTLP) for 
```

Comment:
```suggestion
Promscale natively supports the OpenTelemetry protocol (OTLP) for 
```

Reply:
This should be always referred as `OpenTelemetry Line Protocol`